Dockerfile

```diff
@@ -1,39 +1,40 @@
 # Base
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS base
-WORKDIR /app
-ENV ASPNETCORE_ENVIRONMENT=production
+ARG REGISTRY=562955126301.dkr.ecr.eu-west-2.amazonaws.com
 
-# Development
-FROM base AS development
-WORKDIR /FFCDemoPaymentService
-RUN apt-get update \
-  && apt-get install -y --no-install-recommends unzip \
-  && curl -sSL https://aka.ms/getvsdbgsh | bash /dev/stdin -v latest -l /vsdbg
-COPY ./FFCDemoPaymentService/*.csproj ./
+# FROM $REGISTRY/ffc-dotnet-parent/sdk:$BASE_VERSION AS 
+FROM dotnet-sdk-parent as build
+RUN mkdir -p /app/FFCDemoPaymentService/
+WORKDIR /app/FFCDemoPaymentService/
+COPY --chown=www-data:www-data ./FFCDemoPaymentService/*.csproj ./
+RUN ls -la
 RUN dotnet restore
-COPY ./FFCDemoPaymentService ./
-ENTRYPOINT [ "dotnet", "watch", "run", "--urls", "http://0.0.0.0:3007" ]
+COPY --chown=www-data:www-data ./FFCDemoPaymentService/ ./
+RUN dotnet publish -c Release -o /app/out
+
+# Development
+FROM build AS development
+USER root
+RUN apk update \
```

Comment:
Could this stage be moved to the parent development image?  We'd always want a way to debug .Net Core in a container.